engine/evd.ml

```diff
@@ -948,20 +948,18 @@ let mkLEvar sigma (evk, args) =
   let args = List.fold_right2 fold (evar_filtered_context evi) args SList.empty in
   mkEvar (evk, args)
 
+let is_relevance_irrelevant sigma r =
+  match UState.nf_relevance sigma.universes r with
+  | Irrelevant -> true
+  | Relevant -> false
+  | RelevanceVar _ -> false (* Should be an anomaly *)
```

Comment:
The comment is wrong: there can be undefined sort variables.

Reply:
I changed it, but I'm gonna wait for more significant changes before pushing

Reply:
Ping. The rest of the PR is fine to me.